### DISCUSSION TITLE
Issues with verification.

### DISCUSSION THREAD

**Discussion by CodecNomad** [community/community#51710] [GitHub Education]
Hi! Been having issues with it. 1. The prechecks are messing me up and not even letting my apply, is there a way to bypass them? 2. How can I contact a human and not the virtual assistant if that's not possible.

**Comment by tusharhqq** [comment]
Can you please provide me with more details about the prechecks that are causing issues? Depending on the type of prechecks, there may be ways to bypass them, but it's important to understand the nature of the prechecks before proceeding.

Regarding your second question, you can contact a human representative from the company by visiting their official website and looking for their contact information. This may include an email address, phone number, or live chat support. Additionally, some companies have a customer support page that offers a support ticket system or a knowledge base that you can use to find solutions to common issues.

**Reply by CodecNomad** [reply]
I opened a ticket for an other reason but I'll try contacting them there, also It's the precheck that says you have a 3rd party thing etc.

**Reply by samus-aran** [reply]
> Regarding your second question, you can contact a human representative from the company by visiting their official website and looking for their contact information. This may include an email address, phone number, or live chat support. Additionally, some companies have a customer support page that offers a support ticket system or a knowledge base that you can use to find solutions to common issues.

We have recently made the announcement that Education support will not process individual manual reviews. Please [check here](https://github.com/orgs/community/discussions/49366) for more details on this change.

**Comment by samus-aran** [comment]
Hi @NormiDev 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction. 👍

**Reply by CodecNomad** [reply]
I got it already

**Reply by samus-aran** [reply]
Oh that's awesome news! 🎉 What did you do differently sharing that may help others in similar situations.

**Reply by CodecNomad** [reply]
I already sent that In another discussion, I don't really remember but should be found there.
